### ISSUE TITLE
Icons not visible when using Windows High Contrast Mode

### ISSUE THREAD

**Issue by sirrah-tam** [ithaka/pharos#688]
**Expected behavior**
While using the High Contrast mode on Windows, the Pharos icon's are visible and convey what type of element they are associated with, ie button, link, or static content.

**Actual behavior**
The fill of the SVG is set to black which when used with a black background makes it so it doesn't not appear visually. 

**Steps to reproduce the issue**

1. While using a Windows machine, enable the High Contrast mode (High Contrast 1, High Contrast 2, and High Contrast Black)
2. Navigate to [Storybook | Icons | Names](https://pharos.jstor.org/storybooks/wc/?path=/story/components-icon--names) or any page that uses Pharos icons
3. Verify icon does not appear visually.

**Screenshots or code**
![HCM item page storybook](https://github.com/ithaka/pharos/assets/6874453/0c13a645-bf28-4c98-b2c4-7a99d0a7e2b8)
![HCM icons storybook](https://github.com/ithaka/pharos/assets/6874453/f0f8e8b3-de0b-414d-bd58-40988ae512bf)

**Pharos version**
Pharos 13.4.0

**Your environment**

- OS: Windows
- Browser: Chrome, Chromium Edge, Firefox
- Version: 121.0.6, 121.0.2, 123.0

**Additional information**
Working on a fix for this using the media query `forced-colors`

References:
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors
https://www.smashingmagazine.com/2022/06/guide-windows-high-contrast-mode/
https://css-tricks.com/accessible-svgs-high-contrast-mode/
